pkg/fieldpath/fieldpath.go

```diff
@@ -22,6 +22,15 @@ import (
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/api/meta"
 )
 
+// formatMap formats map[string]string to a string.
+func formatMap(m map[string]string) string {
+	var l string
+	for key, value := range m {
+		l += key + "=" + value + "\n"
```

Comment:
This is key=value rather than json or yaml? Why was this choice made? Feel free to link to the comment thread if this was already discussed.

We may want to add an output format option in the future.